From 522a291e9bd68bb680aa4dbda1d79cca5203833b Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Mon, 17 Sep 2018 17:48:59 +0800
Subject: [PATCH] arm64: configs: rk3399pro_npu_defconfig: enable usb

This patch enables usb dwc3 gadget, configfs,
functionfs and usb 2.0 phy.

Change-Id: I194c7464286083426740aa82a907cc4aa0cb57b3
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 arch/arm64/configs/rk3399pro_npu_defconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/configs/rk3399pro_npu_defconfig b/arch/arm64/configs/rk3399pro_npu_defconfig
index 26f847deaff9..5326af3cf09d 100644
--- a/arch/arm64/configs/rk3399pro_npu_defconfig
+++ b/arch/arm64/configs/rk3399pro_npu_defconfig
@@ -90,6 +90,11 @@ CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_SYR82X=y
 # CONFIG_HID_GENERIC is not set
+CONFIG_USB_DWC3=y
+CONFIG_USB_GADGET=y
+CONFIG_USB_CONFIGFS=y
+CONFIG_USB_CONFIGFS_F_FS=y
+CONFIG_USB_CONFIGFS_UEVENT=y
 CONFIG_DMADEVICES=y
 CONFIG_PL330_DMA=y
 CONFIG_STAGING=y
@@ -114,6 +119,7 @@ CONFIG_ROCKCHIP_SUSPEND_MODE=y
 CONFIG_PM_DEVFREQ=y
 CONFIG_DEVFREQ_GOV_USERSPACE=y
 CONFIG_PM_DEVFREQ_EVENT=y
+CONFIG_PHY_ROCKCHIP_INNO_USB2=y
 CONFIG_PHY_ROCKCHIP_PCIE=y
 CONFIG_ANDROID=y
 CONFIG_ROCKCHIP_SIP=y
-- 
2.35.3

